### ISSUE TITLE
Unable to get text from shape and mathematical formula.

### ISSUE THREAD

**Issue by Laxmi530** [python-openxml/python-docx#1106]
I am trying to read the docx file which I am able to do but I am not getting whole text in the docx file. It is missing the mathematical formula and the text inside the shape. Below is the sample docx file and the code screenshot.

![image](https://user-images.githubusercontent.com/76505459/175287817-f1f98683-8e38-45d6-afc2-b22222f520e8.png)

[Sample.docx](https://github.com/python-openxml/python-docx/files/8966770/Sample.docx)

**Comment by anil1712** [comment]
@Laxmi530 I am trying the same, did you get any solution regarding the same?

**Comment by Laxmi530** [comment]
yes able to get the text from shape but not the mathematical formula

**Comment by Cerebex** [comment]
> yes able to get the text from shape but not the mathematical formula

How did you accomplish getting text from shapes?
